tests/flow/test_ts_madd.py

```diff
@@ -65,7 +65,7 @@ def test_partial_madd():
 
         now = int(time.time() * 1000)
         res = r.execute_command("ts.madd", 'test_key1', "*", 10, 'test_key2', 2000, 20, 'test_key3', 3000, 30)
-        assert now <= res[0] and now + 2 >= res[0]
```

Comment:
@danni-m  the second condition might not be true if within valgrind takes more than 2 seconds to execute ts.madd. Removed this condition given it was flaky